src/Ssh/Ssh/Models/PSSshConfigEntry.cs

```diff
@@ -64,6 +64,11 @@ public class PSSshConfigEntry
         /// </summary>
         public string Port { get; set; }
 
+        /// <summary>
+        /// Resource tag.
+        /// </summary>
+        public string ResourceTag { get; set; }
```

Comment:
I don't see the need to add the resource tag to the config entry. The goal of this class is basically to just get a bunch of values and use them to create the config file string. All it needs to create the config is the value of the port. Any logic to set the value of the port using the tags should be outside of this class.